### DISCUSSION TITLE
Why should I use this library?

### DISCUSSION THREAD

**Discussion by camohub** [php-webdriver/php-webdriver#1119] [General]
Hi, why should I use this library if you use Facebook driver https://github.com/php-webdriver/php-webdriver/blob/main/example.php 
What is the benefit if this lib? Also have you any example how to handle errors. For example can I store errors in databse or can I make a screenshots and store them in a disc?

**Comment by ernstki** [comment]
@camohub This *is* the Facebook WebDriver. It's the same library, but the repository was moved; you can prove this to yourself by visiting https://github.com/facebook/php-webdriver and seeing that it redirects back here.

This is mentioned [in the README](https://github.com/php-webdriver/php-webdriver?tab=readme-ov-file#upgrade-from-version-180).

As for your other questions, the answers are **yes** and **yes**. You can technically store whatever you want in a database, and yes, there is a way to [take screenshots](https://github.com/php-webdriver/php-webdriver/wiki/Example-command-reference#screenshots) with this library.

I can't personally provide you solutions, because I'm still learning to use this library myself. But I recommend changing the title of your question and expounding a bit on what it is you're looking for. Otherwise, it comes across like you're asking random people on the Internet to do your homework for you. :)
